home/web/client/index.html

```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Parity Home</title>
     <script src="./inject.js"></script>
+    <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
Wouldn't it be better to load the scripts from main file? (to have it processed by webpack?)
Right now it might not be copied to build directory also.